guides/v2.1/architecture/gdpr/magento-2x.md

```diff
@@ -229,16 +232,21 @@ The following columns in the `quote` table contain customer information:
 
 Column | Data type
 --- | ---
-`customer_id` | int(10)
-`customer_email` | varchar(255)
-`customer_prefix` | varchar(40)
-`customer_firstname` | varchar(255)
-`customer_middlename` | varchar(40)
-`customer_lastname` | varchar(255)
 `customer_dob` | datetime
+`customer_email` | varchar(128)
+`customer_firstname` | varchar(128)
```

Comment:
Please, make sure that the field sizes are correct here and everywhere. Take a look at 
https://github.com/magento/magento2/blob/1f9186c3b9a96c5e642fd5d3d31ac5c7e1877d2b/app/code/Magento/Quote/Setup/InstallSchema.php#L179
Wher the customer_firstname is VARCHAR(255)

Reply:
@rogyar Corrected data types of fields like customer_firstname, customer_lastname, etc..